stable/risk-advisor/README.md

```diff
@@ -0,0 +1,20 @@
+# risk-advisor
```

Comment:
Please follow the standard readme style. See e. g. https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/README.md